assets/admin/editor-wizard/use-editor-wizard-steps.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
+import { store as editorStore } from '@wordpress/editor/build/store';
+
+/**
+ * Internal dependencies
+ */
+import CourseDetailsStep from './steps/course-details-step';
+import CourseUpgradeStep from './steps/course-upgrade-step';
+import CoursePatternsStep from './steps/course-patterns-step';
+import LessonDetailsStep from './steps/lesson-details-step';
+import LessonPatternsStep from './steps/lesson-patterns-step';
+import { EXTENSIONS_STORE } from '../../extensions/store';
+
+/**
+ * Returns the list of components (representing steps) for the Editor Wizard according to the post type and to if
```

Comment:
```suggestion
 * Returns the list of components (representing steps) for the Editor Wizard according to the post type and if
```

I think the "to" shouldn't be here, right?